testing_and_setup/compass/manage_regression_suite.py

```diff
@@ -262,7 +262,7 @@ def setup_suite(suite_tag, work_dir, model_runtime, config_file, baseline_dir,
     regression_script.write("        runtime = np.ceil(float(subprocess."
                             "check_output(\n"
                             "                ['grep', 'real', outputfile])."
-                            "split('\\n')[-2].split()[1]))\n")
+                            "decode().split('\\n')[-2].split()[1]))\n")
```

Comment:
@mark-petersen, this will need to be in a separate PR to `develop`.  We're not really allowed to change the basic COMPASS scripts in `ocean/develop`.

Reply:
Also, I really prefer to be explicit about what format is being decoded, i.e. `decode('utf-8')`.